### PR TITLE
Reverted MDCButton changes from Component Themer change:

### PR DESCRIPTION
**Pull request by randallli**
https://github.com/material-components/material-components-ios/pull/1443/files/23191e81201906ef200554ff8a1fa955ae2bc7ed#diff-f89cec76e933099b540c38f711df8c53

### CODE REVIEW

File: components/Buttons/src/MDCButton.m [hunk 1]
```diff
@@ -262,6 +256,10 @@ - (void)commonMDCButtonInit {
   // Block users from activating multiple buttons simultaneously by default.
   self.exclusiveTouch = YES;
 
+  // Default background colors.
+  [self setBackgroundColor:MDCColorFromRGB(MDCButtonDefaultBackgroundColor)
+                  forState:UIControlStateNormal];
+
```
Comment (jgunaratne) [inline components/Buttons/src/MDCButton.m:262]: From what I understand this revert will cause the button color to always be set to the default color when commonMDCButtonInit is called. This means the button background color property will never be nil and the UIAppearance proxy color will never be used. We expect clients to be able to use a UIAppearance proxy from another view controller to set colors for all buttons. Setting background color here will prevent the UIAppearance proxy color from being read.

**Review by jgunaratne — CHANGES_REQUESTED** [review]
Need to resolve issue with UIAppearance proxy color never being read when set from a view controller.

File: components/Buttons/src/MDCButton.m (hunk 1 above)
Comment (ianegordon) [reply components/Buttons/src/MDCButton.m:262]: That is my understanding as well.  Let's all discuss in person.

**Review by ianegordon — CHANGES_REQUESTED** [review]

**Comment by ianegordon** [comment]
As a temporary fix to unblock the release, SGTM.
I'll open an issue to identify move the default values back to +initialize / determine what went wrong.

**Comment by ianegordon** [comment]
Opened #1492 to track the issue.

**Review by ianegordon — APPROVED** [review]

**Review by jgunaratne — APPROVED** [review]
lgtm for the release, but we should discuss a strategy to support themers.

**Pull request merged**